stack.yaml

```diff
@@ -0,0 +1,68 @@
+# This file was automatically generated by 'stack init'
```

Comment:
I don't use Stack myself, and don't want to make using it a requirement for users of this package. We ought to be compatible with it, but I'm not willing to maintain a Stack configuration for this package myself. Much of what you've added in this patch seems to assume that Stack is being used.

Reply:
Thank you very much for your many replies.

I had a mistake. I don't want to create pull-request 69c9a4f and 3f96ca1. But I did accidentally . These two commits are related to my solution for the issue #16. But I don't want to create pull-request. So please ignore these two commits.